src/rideshare/settings.py

```diff
@@ -154,3 +154,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+FILE_UPLOAD_MAX_MEMORY_SIZE = 15000000
```

Comment:
```suggestion
FILE_UPLOAD_MAX_MEMORY_SIZE = 1024 * 1024 * 15
```
This is convention and makes the number of MB more clear.

Reply:
addressed [here](https://github.com/cuappdev/scoop-backend/pull/49/commits/d03fddf0358c002bb3a8843d2309bcaddebbaa5e)